.gitignore

```diff
@@ -1,52 +1 @@
-# Compiled source #
-###################
-*.com
-*.class
-*.dll
-*.exe
-*.o
-*.so
-
-# Packages #
-############
-# it's better to unpack these files and commit the raw source
-# git has its own built in compression methods
-*.7z
-*.dmg
-*.gz
-*.iso
-*.jar
-*.rar
-*.tar
-*.zip
-
-# Logs and databases #
-######################
-*.log
-*.sql
-*.sqlite
-
-# OS generated files #
```

Comment:
I think I would keep some of those 

.DS_Store
.DS_Store?
._*
.Spotlight-V100
.Trashes
ehthumbs.db
Thumbs.db
.vscode

Reply:
Are any changes to this file necessary? They seem out of scope for the current PR and may be accidentally.

Reply:
@DL6ER This PR causes the `piHoleVersion` to be created in the current directory.

Since `piHoleVersion` should not be tracked by git, therefore it is necessary to add it in the `.gitignore`, right?

Reply:
Yes, adding this one file but not removing all others. It's best to keep PRs as narrow as possible, ideally doing a single thing (and everything that is related). "Cleaning" the `.gitignore` seems out of place is what I wanted to say.

Reply:
@DL6ER Ah.. okay. Will fix it soon when I'm back at my workstation.

Reply:
Fixed.